ddtrace/contrib/langchain/patch.py

```diff
@@ -706,11 +722,18 @@ def traced_lcel_runnable_sequence(langchain, pin, func, instance, args, kwargs):
     This method captures the initial inputs to the chain, as well as the final outputs, and tags them appropriately.
     """
     integration = langchain._datadog_integration
-    span = integration.trace(pin, "%s.%s" % (instance.__module__, instance.__class__.__name__), interface_type="chain")
+    span = integration.trace(
+        pin,
+        "{}.{}".format(instance.__module__, instance.__class__.__name__),
+        submit_to_llmobs=True,
+        interface_type="chain",
+    )
+    raw_inputs, inputs = None, None
+    final_output = None
     try:
-        inputs = get_argument_value(args, kwargs, 0, "input")
+        raw_inputs = inputs = get_argument_value(args, kwargs, 0, "input")
```

Comment:
ah yeah no it can be the same now, good catch. had it this way before for when I was conditionally setting/formatting the inputs for either `INPUT_VALUE` or `INPUT_PARAMETERS`. now that we're just stringifying for `INPUT_PARAMETERS`, it doesn't matter, thanks for the catch!